dsaps/models.py

```diff
@@ -123,6 +125,28 @@ class MetadataEntry(BaseRecord):
     language = Field()
 
 
+def build_file_list_local(directory, file_extension):
+    """Build list of files in local directory."""
+    file_list = {}
+    for root, dirs, files in os.walk(directory, topdown=True):
+        for file in files:
+            if file.endswith(file_extension):
+                full_file_path = os.path.join(root, file).replace('\\', '/')
```

Comment:
What's the reason for using the replace here? I'm assuming it has something to do with Windows, but I'd like to better understand the requirement this is solving.

Reply:
I think this is legacy code from my last job that I copied without thinking, it doesn't seem necessary so I'll remove it

Reply:
In this case, I would just get rid of this function entirely. The standard library already provides the ability to recursively generate a list of files matching a specific extension:

```python
import glob

files = glob.glob('/path/to/search/**/*.pdf', recursive=True)
```

If there are potentially going to be lots (thousands) of files I'd suggest using `iglob` instead. This functionality is also in the [pathlib](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob) module.

Reply:
Makes sense, I will save this for when I create a click command for the full workflow. Thanks!